### PR TITLE
initial skeleton for driver & accept trip request

### PR DESCRIPTION
**Pull request by loaighoraba**

### CODE REVIEW

File: fleeto_backend/app/services/drivers/trip_requests/accept_service.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  def initialize(driver, trip_request)
+    @driver = driver
+    @trip_request = trip_request
+  end
+
+  def execute
+    @trip_request.update(status: TripRequest::ACCEPTED)
+    Trip.create!(
+      customer: @trip_request.customer,
+      driver: @driver, from_long: trip_request,
```
Comment (mohelsaka) [inline fleeto_backend/app/services/drivers/trip_requests/accept_service.rb]: from_long: trip_request,

This sounds wrong.

File: fleeto_backend/app/controllers/customer/v1/trip_requests_controller.rb [hunk 2]
```diff
@@ -14,8 +14,7 @@ def create
 
   def destroy
```
Comment (mohelsaka) [inline fleeto_backend/app/controllers/customer/v1/trip_requests_controller.rb]: This should explicitly be named "canceled" and the service name should be TripRequets::CancelationService.
Destroy is kinda implicit.

This is just an opinion.

File: fleeto_backend/app/models/trip_request.rb [hunk 3]
```diff
@@ -1,10 +1,14 @@
 class TripRequest < ApplicationRecord
   belongs_to :customer
-
+  has_one :trip
+  
   validates :from_lat, :from_long, presence: true
 
   validate :customer_is_not_in_a_trip
 
+  PENDING, CANCELED, ACCEPTED = STATUSES = %w(pending canceled accepted)
```
Comment (loaighoraba) [inline fleeto_backend/app/models/trip_request.rb]: I think expired will be a better name

File: fleeto_backend/app/services/drivers/trip_requests/accept_service.rb (hunk 1 above)
Comment (loaighoraba) [inline fleeto_backend/app/services/drivers/trip_requests/accept_service.rb]: correct :)

File: fleeto_backend/app/models/trip_request.rb (hunk 3 above)
Comment (mohelsaka) [inline fleeto_backend/app/models/trip_request.rb]: Agreed.

**Pull request merged**